frontend/src/components/pages/login/Login.tsx

```diff
@@ -1,28 +1,35 @@
-import React from 'react';
+import Box from '@material-ui/core/Box';
+import axios from 'axios';
+import React, { useState } from 'react';
+import { GoogleLogin } from 'react-google-login';
+import { useHistory } from 'react-router-dom';
 import GoogleLogo from '../../../images/google-login.svg';
 import Logo from '../../../images/logo-login.svg';
+import { getUserByMail } from '../../../services/apiUserService';
 import { saveUserToStorage } from '../../../services/localStorageService';
-import user from '../../mocks/user-mock.json';
+// import user from '../../mocks/user-mock.json';
 import Footer from '../../shared/footer/Footer';
 import Tile from '../../shared/tile/Tile';
 import './login.scss';
-import Box from '@material-ui/core/Box';
-import { Link } from 'react-router-dom';
-import { GoogleLogin } from 'react-google-login';
-import axios from 'axios';
 
 const url = 'http://localhost:8000';
 
-function responseGoogle(response: any): void {
-  axios({
-    method: 'POST',
-    url: `${url}/api/googleLogin`,
-    data: { tokenID: response.tokenId }
-  });
-  saveUserToStorage(user);
-}
-
 const Login: React.FC = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const history = useHistory();
+  function responseGoogle(response: any): void {
+    axios({
```

Comment:
move ALL axios usage into services

No axios should be used in components code